### PR TITLE
added mdx with images to folder

### PR DESCRIPTION
**Pull request by saminacodes**

### CODE REVIEW

**Comment by nachoiacovino** [comment]
First, you need to create the learn-web3 folder, and not have this inside the learn-thirdweb

File: portal/learn-thirdweb/intro-to-web3.mdx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+authorAvatar: "/assets/portal/authors/odyssey.png"
+authorTwitter: "https://twitter.com/odyssey_dao"
+draft: false
+summary: ""
+image: ""
+date: "2022-02-06"
+updated: ""
+---
+
+# Level 0 - Version 3
```
Comment (nachoiacovino) [inline portal/learn-thirdweb/intro-to-web3.mdx]: Never one #, this is an H1 and we should never have them.

File: portal/learn-thirdweb/intro-to-web3.mdx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+Blockchains are:
+
+1. **Decentralized**: Transactions are on a network of computers (nodes).
+2. **Immutable**: Transactions cannot be changed once committed.
+3. **Open**: Transactions can be viewed by anyone.
+
+Blockchain is **a system of recording information in a way that makes it difficult or impossible to change, hack, or cheat the system**. A blockchain is essentially a digital ledger of transactions that is duplicated and distributed across the entire network of computer systems on the blockchain.
+
+## **How does Ethereum work?**
```
Comment (nachoiacovino) [inline portal/learn-thirdweb/intro-to-web3.mdx]: This is what breaking the build, can't have ** in headings

**Review by nachoiacovino — CHANGES_REQUESTED** [review]
Move to the correct folder, fix headings and the build

**Comment by nachoiacovino** [comment]
Please also move the assets to the correct folder and reference them correctly

Also, if you feel capable, add this to the sidebar and fix the navigation of the previous/next pages too, otherwise I can do it tomorrow night

**Comment by saminacodes** [comment]
Not sure how to do that. Would be more comfortable with you doing it this time. Could I observe the process so I know for next time?

**Comment by nachoiacovino** [comment]
Please have a parent folder called learn-web3, also I feel like this is the landing for /learn-web3 right? In that case the file should be called index

That means images should be under assets/learn-web3/index/x.png

Also, make sure to have descriptive images names, not 1,2,3.

And make sure to slugify the image names.  

Descriptive names for images help with SEO.

**Pull request merged**